.github/workflows/workflow-bump-version.yaml

```diff
@@ -0,0 +1,77 @@
+name: Bump version
+
+on:
+  workflow_call:
+    inputs:
+      new-maven-version:
+        description: 'New maven version'
+        type: string
+        required: false
+  workflow_dispatch:
+    inputs:
+      new-maven-version:
+        description: 'New maven version'
+        type: string
+        required: false
+
+jobs:
+  bump-versions:
+    name: Bumps versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: PREP / Checkout code
+        uses: actions/checkout@v4
+      - name: PREP / Setup git
+        run: |
+          git config --global user.email "miranum@miragon.de"
+          git config --global user.name "Miranum Github Bot"
+      - name: PREP / Prepare mvnw
+        run: chmod +x ./mvnw
+      - name: PREP / Install Java and Maven
+        if: inputs.bump-services-version== true
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'corretto'
+
+      - name: BUMP VERSION / Raise mvn version
+        if: inputs.new-maven-version != ''
+        run: |
+          ./mvnw versions:set --batch-mode -DprocessAllModules -DnewVersion=${{ inputs.new-maven-version }}-SNAPSHOT -DprocessAllModules
+          ./mvnw versions:commit -DprocessAllModules
+      - name: BUMP VERSION / Raise mvn version
+        if: inputs.new-maven-version == ''
+        run: |
+          ./mvnw build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.nextMinorVersion}.0-SNAPSHOT -DprocessAllModules
+          ./mvnw versions:commit -DprocessAllModules
```

Comment:
This overwrites the `${revision}` with the new version. How should we auto version bump maven after releases?

Reply:
@DaAnda97 how do you usually do version bumps with the maven revision?

Reply:
the version usually is MAJOR.MINOR.PATCH
I used to to that manually yet

Reply:
Should we stick to manually updating the version or should we use the auto version bump and overwrite all `${revision}`?

Reply:
I'd prefere an automatic increase